recipes_source/recipes/README.txt

```diff
@@ -2,7 +2,7 @@ PyTorch Recipes
 ---------------------------------------------
 1. loading_data_recipe.py
 	 Loading Data in PyTorch
-         https://pytorch.org/tutorials/recipes/recipes/loading_data_recipe.html
+         https://pytorch.org/tutorials/recipes/loading_data_recipe.html
```

Comment:
just delete this entry.

Reply:
Shouldn't this point to the Datasets & DataLoaders tutorial instead of being removed completely?

Reply:
No, this file needed for sphinx gallery. You can delete that entry.